graphql/resolvers.go

```diff
@@ -2871,13 +2871,26 @@ func (r *taskResolver) IsPerfPluginEnabled(ctx context.Context, obj *restModel.A
 				return false, err
 			}
 			for _, projectName := range perfPlugin.Projects {
-				if projectName == pRef.Id || projectName == pRef.Identifier {
-					return true, nil
+				if projectName != pRef.Id || projectName != pRef.Identifier {
+					return false, nil
 				}
 			}
 		}
+		opts := apimodels.GetCedarPerfCountOptions{
+			BaseURL: evergreen.GetEnvironment().Settings().Cedar.BaseURL,
+			TaskID:  *obj.Id,
```

Comment:
Can this ever be the ID of a task in the old collection? Cedar does not recognize those, we always have to use the task id from the new collection (which is stored in old tasks under the field `old_task_id`).

Reply:
Not sure. Maybe @khelif96 knows, if not I can try and figure it out.

Reply:
No it shouldn't but that is a good point. We should include execution in this request. Since the existence of perf data may change on a per execution basis.